public/views/index-template.html

```diff
@@ -255,6 +255,9 @@
         }
       };
 
+      // Adapt document language
+      document.documentElement.lang = window.grafanaBootData.user.language
```

Comment:
Could we set this directly on `<html lang="en">` instead?

Reply:
You are right! I got confused. I updated the commit.